core/services/job/models.go

```diff
@@ -886,7 +896,6 @@ func (w *WorkflowSpec) Validate() error {
 	}
 	w.WorkflowOwner = strings.TrimPrefix(s.Owner, "0x") // the json schema validation ensures it is a hex string with 0x prefix, but the database does not store the prefix
 	w.WorkflowName = s.Name
-	w.WorkflowID = s.CID()
```

Comment:
why is this removed?

Reply:
The WASM shouldn't report the CID.  The host should be responsible for it.  @cedric-cordenier and I were talking, I'll move the calculation here.



I think I have more work to do.